blocks/site-details/site-details.js

```diff
@@ -1272,12 +1266,13 @@ export default async function decorate(block) {
           renderTable(block.querySelector('.emails tbody'), emails, 'emails');
         })
         .catch((error) => {
+          // eslint-disable-next-line no-console
           console.log(error);
         });
 
       // MARK: add page button
       block.querySelector('.add-page').onclick = () => {
-        addPageDialogSetup({ project, headers, darkAlleyVariation });
+        addPageDialogSetup({ project, headers });
```

Comment:
Why did you remove `darkAlleyVariation` ?

Reply:
#21 should have added it to my PR, but somehow it wasn't applied.